src/FixerDefinition/VersionSpecification.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\FixerDefinition;
+
+/**
+ * @author Andreas Möller <am@localheinz.com>
+ *
+ * @internal
+ */
+final class VersionSpecification implements VersionSpecificationInterface
+{
+    /**
+     * @var int|null
+     */
+    private $minimum;
+
+    /**
+     * @var int|null
+     */
+    private $maximum;
```

Comment:
@SpacePossum you have asked for it, @localheinz you have accept that request.
but my question was not been aswered

Why we need to have maximum PHP version ?
We do not have any fixer that would specify that